### ISSUE TITLE
Email sending on Windows 7: SSL Exception: The function requested is not supported

### ISSUE THREAD

**Issue by erl987** [erl987/personalfme#5]
# Affected version
PersonalFME 1.0.0

# Operating system
Windows 7

# Problem
When sending emails under Windows 7 using the provider GMX, the following error is occurring (and email sending is not working):

```
SSL Exception: Error during handshake: The function requested is not supported
```

**Comment by erl987** [comment]
# Probable reason
Under Windows PersonalFME is using SChannel (the Windows implementation of SSL/TLS). Windows 7 had end of support on 14-01-2020. Apparently, the GMX server is requiring SSL/TLS features that are not supported by SChannel on Windows 7. The does probably affect other Email servers as well.

# Actions
OpenSSL cannot be used by PersonalFME under Windows due to license constraints. Windows 7 will be removed from the list of supported OS in the next release of PersonalFME.

**Comment by erl987** [comment]
This will be fixed in PersonalFME 1.0.1.
